packages/demo-cards/author.gts

```diff
@@ -21,7 +22,15 @@ export class Author extends Card {
 
   static embedded = class Embedded extends Component<typeof this> {
     <template>
-      <@fields.firstName /> <@fields.lastName />
+      <CardContainer
+        class='demo-card'
+        @displayBoundaries={{true}}
+        {{! @glint-ignore  Argument of type 'unknown' is not assignable to parameter of type 'Element'}}
+        ...attributes
```

Comment:
I'm struggling with this lint error. Do perhaps we have an example somewhere how to allow `...attributes` in component signatures?